scripts/pi-hole/js/db_graph.js

```diff
@@ -44,7 +38,7 @@ $(function () {
           moment().subtract(1, "month").endOf("month"),
         ],
         "This Year": [moment().startOf("year"), moment()],
-        "All Time": [moment(mintimestamp), moment()],
```

Comment:
@yubiuser 
Why this was removed?

Are you reverting #1969 and #1976 ?

Reply:
Fixed.